File: C++/7thOct-2.cpp
#include <iostream>
#include<string>
using namespace std;
string inwords(long n){
    string res="";
    
    if(n/10000000){
        res+=to_string(n/10000000)+" crores ";
        n=n%10000000;
    }    
    if(n/100000){
    res+=to_string(n/100000)+" lacs ";
    n=n%100000;
        
    }
    if(n/1000){
        res+=to_string(n/1000)+" thousands ";
        n=n%1000;
    }
    if(n/100){
        res+=to_string(n/100)+" hundreds ";
        n=n%100;
    }
    if(n/10){
        res+=to_string(n/10)+" tens and ";
        n=n%10;
    }
    
    res+=to_string(n);
    
    return res;
}

int main()
{   long n;
    cin>>n;
    cout<<inwords(n);
    return 0;
}

File: Python/7thOct-2.py
def number(num):
	l=[]
	while num>0:
		m=num%10
		num= num//10
		l.append(m)
	print(l)
	name(l)

def name(s):
	n1=[" "," Ten and "," Hundred "," Thousand ",""," Lac ",""," Crore ","",]
	n2=[" "," Tens and "," Hundreds "," Thousands ",""," Lacs ",""," Crores ","",]
	st=''
	for i in range(len(s)-1, -1 ,-1):
		if s[i]>1:
			st=(st+str(s[i])+n2[i])
		else:
			st=(st+str(s[i])+n1[i])
	print(st)

	
n=int(input("Enter the Number : "))
number(n)